### PR TITLE
Issue423 durack1 add vacumm package

### PR DESCRIPTION
**Pull request by durack1**
@doutriaux1 take a look at this in response to #423

### CODE REVIEW

**Comment by durack1** [comment]
@doutriaux1 it seems like this is being prompted to build:
`[INFO] These are the packages that we will try to build: basemap;CDAT;CURL;Cython;FFMPEG;g2clib;gdal;GEOS;gui_support;h5py;HDF5;ZMQ;pyzmq;tornado;IPYTHON;jasper;jpeg;LAPACK;lepl;libcf;Matplotlib;SIX;DATEUTILS;PYPARSING;Mpi4py;PYOPENSSL;setuptools;MyProxyClient;NetCDF;NetCDFPLUS;Numexpr;NUMPY;MPI;OSMesa;Python;**vacumm**;VTK;pkgconfig;png;pyspharm;PyTables;readline;R;scikits;SCIPY;pip;Shapely;pygments;markupsafe;jinja2;docutils;Sphinx;freetype;zlib;libXML2;libXSLT;png;pixman;fontconfig;tiff;Curses;CMOR;udunits2;uuid;UVCMETRICS;vistrails;YASM;PYLIBXML2;CDATLogger;pyclimate;scientificpython;windspharm;eof2;eofs;windfield`

**Comment by doutriaux1** [comment]
@durack1 you need to make them dependent on python at least

**Comment by durack1** [comment]
@doutriaux1 isn't that achieved in the vacumm_deps.cmake file `set(vacumm ${vacumm_pkg} ${python_pkg})`

File: CMake/cdat_modules/vacumm_deps.cmake [hunk 1]
```diff
@@ -0,0 +1 @@
+set(vacumm ${vacumm_pkg} ${python_pkg})
```
Comment (doutriaux1) [inline CMake/cdat_modules/vacumm_deps.cmake]: @durack1 should be
```
set(vacumm_deps  ${python_pkg})
```

Comment (durack1) [inline CMake/cdat_modules/vacumm_deps.cmake]: Ok well if that's the case then h5py will likely also need a tweak: https://github.com/UV-CDAT/uvcdat/blob/master/CMake/cdat_modules/h5py_deps.cmake#L1

**Comment by aashish24** [comment]
LGTM :+1:

**Comment by durack1** [comment]
@doutriaux1 this PR has been reverted, not sure why it's not showing up here - but #1089 includes all the components to close this

**Comment by stefraynaud** [comment]
vacumm also depends on  pytz.
Since this dependency is marginal, I will certainly remove it and make an import inside the functions that use it.

**Comment by durack1** [comment]
Thanks for the tipoff @stefraynaud I went off the dependency list at http://www.ifremer.fr/vacumm/user.install.prereqalt.html which didn't included pytz - can you double check the dependencies that we currently have in, it's likely these will need to be further revised then..

https://github.com/durack1/uvcdat/blob/issue423_durack1_FixVacummDependencies/CMake/cdat_modules/vacumm_deps.cmake

@aashish24 maybe hold off merging #1089 until this dependency check has been done..

**Comment by stefraynaud** [comment]
@durack1, pytz was previously included in matplotlib, and I just discovered that it's no longer the case. So it becomes an explicit dependency for vacumm for the moment, and its website must be updated. 

Just a remark: when vacumm is installed inside an UVCDAT architecture, a more appropriate list of dependencies is: http://www.ifremer.fr/vacumm/user.install.prereq.html#other-packages

Since pytz adds timezone support to matplotlib and since it is a little package that is useful for geographic information management, perhaps you can include it into uvcdat.

**Comment by stefraynaud** [comment]
Another point: I'm not personally maintaining the vacumm website at IFREMER http://www.ifremer.fr/vacumm
A more uptodate version is the following: http://relay.actimar.fr/~raynaud/vacumm/user.install.prereq.html

**Pull request merged**